crates/bindings-csharp/Runtime/Runtime.csproj

```diff
@@ -41,6 +47,8 @@
     <None Include="build/" Pack="true" PackagePath="build" />
     <None Include="bindings.c" Pack="true" PackagePath="" />
     <None Include="driver.h" Pack="true" PackagePath="" />
+    <!-- We want all users who depends on Runtime to automatically get the Roslyn codegen component as well. -->
+    <None Include="../Codegen/bin/$(Configuration)/netstandard2.0/SpacetimeDB.Codegen.dll" Pack="true" PackagePath="analyzers/dotnet/cs" />
```

Comment:
Again asking for my understanding, feel free to just say "trust me" if it's too much explaining - what's the difference between depending on the `Codegen.csproj` "directly" vs forcing it to build above and then depending on the `dll`?

Reply:
Depending "directly" results in a Runtime nuget package with a dependency on a Codegen nuget package, so we'd still need to pack and publish both on each release.

However, including the DLL of the Codegen package directly in the Runtime one simply "merges" them together so we only need to publish a single package instead.